modules/library/cql/src/test/java/org/geotools/filter/text/ecql/ECQLLiteralTest.java

```diff
@@ -45,40 +46,20 @@ public ECQLLiteralTest(){
      */
     @Test
     public void lineString() throws Exception {
-        Expression resultFilter;
-
-        final String expectedGeometry= "LINESTRING (1 2, 3 4)";
-        resultFilter = CompilerUtil.parseExpression(language, expectedGeometry);
-
-        Assert.assertTrue("DistanceBufferOperator filter was expected",
-                resultFilter instanceof Literal);
-
-        Literal filter = (Literal) resultFilter;
-        Object actualGeometry = filter.getValue();
-        
-        Assert.assertTrue("LineString class was expected",  actualGeometry instanceof LineString);
-
-        assertEqualsGeometries(expectedGeometry, (LineString) actualGeometry);
+        String wkt = "LINESTRING (1 2, 3 4)";
+        assertGeometryParse(wkt, LineString.class, null);
```

Comment:
You could save much duplication with a method `assertGeometryParse(wkt, geometryClass)` that calls `assertGeometryParse(wkt, geometryClass, null)` and `assertGeometryParse(wkt, geometryClass, 4326)`.

And it would also be nice to make `4326` a constant for clarity.

Reply:
Constant being added, refactored out method doing the two calls.